eng/pipelines/templates/steps/vcpkg.yml

```diff
@@ -38,8 +104,14 @@ steps:
         echo "##vso[task.prependpath]$(pwd)"
         echo "##vso[task.setvariable variable=VCPKG_INSTALLATION_ROOT;]$(pwd)"
     # Execute only for Mac and if there is at least one dependency to be installed
```

Comment:
Does this comment need updating since the display name below says `MacOS & Linux` and not `only for Mac`?